src/assets/styles/style.less

```diff
@@ -15,7 +15,7 @@
 @import "../../sidebars/favourites/minichart/minichart.less";
 @import "../../sidebars/star/star.less";
 @import "../../showcase/showcase-changes.less";
-@import "custom-scroll.less";
+@import "../../sidebars/custom-scroll.less";
```

Comment:
Is there any particular reason why this has been moved? I know we're only using the scroll in the sidebar at the minute, but it still seems to me like it's applications are sufficiently abstract/general to warrant being in a high level directory. Also my personal preference is to have file moves like this in a separate PR unless they're directly related/necessary for the change that this PR is making.